src/elements/content-sidebar/AddTaskMenu.scss

```diff
@@ -0,0 +1,29 @@
+@import '../common/variables';
+
+$bsc-AddTaskMenu-width: 256px;
+
+.bsc-AddTaskMenu {
+    padding: 0;
+}
+
+.bsc-AddTaskMenu-menuItem {
+    display: flex;
+    max-width: $bsc-AddTaskMenu-width;
+    min-height: 80px;
+    padding: 20px;
+    white-space: normal;
+}
+
+.bsc-AddTaskMenu-avatar {
+    margin-right: 4px;
+}
+
+.bsc-AddTaskMenu-title {
+    color: $threes;
+    font-size: 13px;
+}
+
+.bsc-AddTaskMenu-description {
+    color: $bdl-neutral-02;
+    font-size: 12px;
```

Comment:
Do we actually have adjacent font sizes that differ by a single pixel? Can you verify whether we can't move one of these up, or down?

Reply:
Will do.

Reply:
Set `font-size: 13px` for both.